index.js

```diff
@@ -4,7 +4,36 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    
+    if( process.argv[2] === undefined ){
+        return 'No operation provided...';
+    }else if( process.argv[2] !== `plus` && process.argv[2] !== `minus` ){
+        return `Invalid operation: ${process.argv[2]}`
+    }else if ( process.argv[3] === undefined ){
+        return `No numbers provided...`
+    }else if( process.argv[2] === `plus` ){
```

Comment:
The code in this else if block and the one on line 25 are very similar with few differences. How can you refactor this code to be less repetitive??